pkg/models/new_discussion_replied_meta.go

```diff
@@ -0,0 +1,67 @@
+package models
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+// NewDiscussionRepliedActivityMeta represents the notification data that is relevant to a new Discussion Reply
+type NewDiscussionRepliedActivityMeta struct {
```

Comment:
This looks pretty good! I'd do a final pass looking at [Figma](https://www.figma.com/file/8MfXzXzmKWv8vBjTasUIkD/MINT---Notifications?type=design&node-id=3472-13198&mode=design&t=q13knZsma5QK1gdV-0) and include any other fields that might be needed. At A glance, I think we might need model_plan_ID to be stored, as well as a resolver to modelPlan.

I added this onto Gary's [PR](https://github.com/CMSgov/mint-app/pull/988) for the reply notification type, I didn't realize it was needed, but it's important for rendering.